nginx_ingress_controller/datadog_checks/nginx_ingress_controller/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-__version__ = '1.3.0'
+__version__ = '1.4.0'
```

Comment:
Let's revert this change, versions and changelog bumps are updated separately.

Reply:
Ah, I see. Reverting this change